framework/widgets/InputWidget.php

```diff
@@ -87,4 +87,25 @@ protected function hasModel()
     {
         return $this->model instanceof Model && $this->attribute !== null;
     }
+
+    /**
+     * Create a HTML input tag
+     *
+     * This will call [[Html::activeInput()]] if the input widget is [[hasModel()|tied to a model]],
+     * or [[Html::input()]] if not.
+     *
+     * @param string $type the type of the input to create.
+     * @return string the HTML of the input field.
+     * @since 2.0.13
+     * @see Html::activeInput()
+     * @see Html::input()
+     */
+    protected function createInputField($type)
```

Comment:
isnt this ambigious as it renders a HTML tag and not actually creating a Field? Perhabs a better name is `renderInput()`